edb/schema/delta.py

```diff
@@ -2477,6 +2477,10 @@ def apply(
 class DeleteObject(ObjectCommand[so.Object_T], Generic[so.Object_T]):
     _delta_action = 'delete'
 
+    #: If True, apply the command only if the object has no referrers
+    #: in the schema.
+    if_unused = struct.Field(bool, default=False)
```

Comment:
I'd rename to `only_if_unused` or even more verbose `apply_only_if_unused`. This isn't the most common thing so verbosity isn't going to slow down coding much.

Reply:
This naming is consistent with other `if_` conditions on commands: `if_not_exists`, `if_exists`, etc.  Should we rename those also?

Reply:
I'd rename to `apply_if_unused`, `apply_if_exists`, etc. /2c

Reply:
OK, I'll follow up separately.